include/openenclave/internal/sgxsign.h

```diff
@@ -35,6 +35,31 @@ oe_result_t oe_sgx_sign_enclave(
     size_t pem_size,
     sgx_sigstruct_t* sigstruct);
 
+/**
+ * Digitally sign the enclave with the given hash using an openssl engine
+ *
+ * This function digitally signs the enclave whose hash is given by the
+ * **mrenclave** parameter. The signing key is given by the **engine_id** and related **key_id**
+ * parameter. If successful, the function writes the signature into the
+ * **sigstruct** parameter (an SGX signature structure).
+ *
+ * @param mrenclave[in] hash of the enclave being signed
+ * @param engine_id[in] Text name of the engine to use
+ * @param key_id[in]    Integer handle for the key to use.
```

Comment:
nit: Inconsistent spacing and capitalization between 48 vs 49

Reply:
done